crates/uv-settings/src/lib.rs

```diff
@@ -670,3 +680,25 @@ fn parse_string_environment_variable(name: &'static str) -> Result<Option<String
         },
     }
 }
+
+/// Parse a integer environment variable.
+fn parse_integer_environment_variable(name: &'static str) -> Result<Option<u64>, Error> {
```

Comment:
I think as written, this will return an error on an empty string instead of `None`?

Reply:
I added test for this